src/support/thread_group.c

```diff
@@ -42,14 +42,15 @@ __thread_run(void *arg)
         WT_IGNORE_RET(__wt_panic(session, ret, "Unrecoverable utility thread error"));
 
     /*
-     * The three cases when threads are expected to stop are:
+     * The four cases when threads are expected to stop are:
```

Comment:
I suggest removing the number from the sentence so that people can just edit the list without remembering to update that too. "The cases when threads are ...:"